server/routes/api.js

```diff
@@ -137,9 +137,9 @@ router.post('/clear', function(req, res) {
 
 router.post('/rename', function(req, res, next) {
 	const name = req.body.name;
+	const id = getScreenIDsForRequest(req);
 	const screen = screens.get(id)[0];
 	const oldName = screen ? screen.name : 'No screen present';
-	const id = getScreenIDsForRequest(req);
```

Comment:
Nice catch, hopefully when I implement Sentry bugs such as this will be surfaced.

I'm still pondering how/why our linter didn't throw on this. It is definitely being accessed in a [temporal dead zone](http://jsrocks.org/2015/01/temporal-dead-zone-tdz-demystified/).